homeassistant/components/ai_task/const.py

```diff
@@ -32,3 +34,6 @@ class AITaskEntityFeature(IntFlag):
 
     GENERATE_DATA = 1
     """Generate data based on instructions."""
+
+    GENERATE_STRUCTURED_DATA = 2
```

Comment:
No need for an extra supported feature. It's folded into `GENERATE_DATA` and we won't allow AI Task entities that cannot do that.